src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/clients/IGDBClient.kt

```diff
@@ -15,7 +15,7 @@ import java.net.http.HttpRequest
 import java.net.http.HttpResponse
 
 @Component
-class IGDBClient {
+class IGDBClient(private val externalAPICall: ExternalAPICall) {
```

Comment:
(Kotlin question) Is this showing ExternalAPICall as a class parameter? Do we want to extend the other class instead?

Reply:
Bump on this